STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.examiner.jpa.JPAExaminerTest.testFindAndSetAttributesManyToMany
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/jpa/JPAExaminerTest.java
CLEANER: de.strullerbaumann.visualee.examiner.jpa.TestGroup100Case0.test260
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/jpa/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 19
NEW CLEANER SIZE: 2
ELAPSED TIME: 65.375
==========================
@@ -80,0 +80,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
